test/test.js

```diff
@@ -112,22 +112,22 @@ function createTest(rawSubTest) {
     // eslint-disable-next-line no-param-reassign
     t.context.cleanup = rawSubTest.cleanup;
 
-    const matchText = function matchText(expected, text) {
-      return createRegTest(expected).test(text);
+    const matchText = function matchText(expected, stripCmd, prop) {
+      return createRegTest(expected[prop]).test(stripCmd[`std${prop}`]);
     };
 
     for (const step of rawSubTest.steps) {
       // eslint-disable-next-line no-await-in-loop
       const cmdOut = await t.context.ct.streamIn(step.in);
       if (step.out) {
-        const stripStdout = stripText(cmdOut.stdout);
-        t.true(matchText(step.out, stripStdout));
+        const stripCmdOut = { ...cmdOut, stdout: stripText(cmdOut.stdout) };
```

Comment:
This is very nice!  It occurs to me that it might be easier to read if we gave up on `power-assert` (sorry...) and just used `t.deepEqual`, something like
```js
const cmdToCompare = {
  stdout: step.out && cmdOut.stdout,
  stderr: step.err && cmdOut.stderr,
  exit: cmdOut.exitCode,
};
const expected =
  stdout: step.out,
  stderr: step.err,
  exitCode: step.exit || 0,
};
t.deepEqual(expected, cmdToCompare);
```

Reply:
Ack (as you say 😎)